examples/change-png-info.rs

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2023 Lukas Schattenhofer <lukas.schattenhofer@ose-germany.de>
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
Please remove. We track authorship via git history and all source code in the repository is MIT OR Apache-2.0 as per the LICENSE files in the root directory